weave-js/src/components/PagePanelComponents/Home/Browse3/inference/DetailTile.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * This is a small box for highlighting key propeties of a model.
+ */
+import React from 'react';
+
+type DetailTileProps = {
+  header: string;
+  children?: React.ReactNode;
+  footer?: string;
+};
+
+export const DetailTile = ({header, footer, children}: DetailTileProps) => {
+  return (
+    <div className="flex h-[96px] w-[150px] flex-col border border-moon-250">
+      <div className="text-center text-sm font-semibold">{header}</div>
+      <div className="flex flex-grow items-center justify-center">
+        {children}
+      </div>
+      <div className="text-center text-sm text-moon-500">
+        {footer ?? '\u00A0'}
```

Comment:
`'\u00A0'` what char is this? Comment?

Reply:
Unicode [non-breaking space](https://www.fileformat.info/info/unicode/char/00a0/index.htm). Will comment.